packages/contracts/script/DeployEmailSignerFactory.s.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+import {BaseDeployScript} from "./BaseDeployScript.sol";
+import {EmailSigner} from "../src/EmailSigner.sol";
+import {EmailSignerFactory} from "../src/EmailSignerFactory.sol";
+
+contract DeployEmailSignerFactory is BaseDeployScript {
```

Comment:
We should get into the habit of writing tests for scripts. Can you add and link a linear issue for adding a test for this script, so we can address this in the future?